thirdparty/embree/common/simd/arm/sse2neon.h

```diff
@@ -102,7 +102,9 @@
 #include <stdint.h>
 #include <stdlib.h>
 
-#if defined(_WIN32)
+// -- GODOT start --
+#if defined(_WIN32) && !defined(__clang__)
```

Comment:
Why doesn't it work on llvm-mingw specifically but works on mingw-w64-gcc?

Reply:
Most likely it's not clang but UCRT specific, but `_mm_malloc` is not found with this define.

Reply:
Also, this is ARM specific code, so MinGW-GCC might have it missing as well.

Reply:
Should we use `!defined(__MINGW32__)` then?

Reply:
Either way, worth PR'ing upstream to get more eyes on it and find out the proper fix there: https://github.com/DLTcollab/sse2neon

Downstream I'm fine with either option to get this to compile.

Reply:
MSVC define it for all architectures, and it's just a macro for the same `_aligned_malloc` I have added, so it's the same.

Reply:
Then maybe it should just be `_MSC_VER`?
Is it defined for clang-cl?

Reply:
No idea, but `__MINGW32__` definitely should not be defined.

Reply:
Upstream PR - https://github.com/DLTcollab/sse2neon/pull/637